### PR TITLE
Import support for map data

### PR DESCRIPTION
**Pull request by Lemongrass3110**
This pull should provide the rAthena users with the possibility to add maps by putting them into import files.

### CODE REVIEW

File: src/common/mapindex.c [hunk 1]
```diff
@@ -134,29 +132,44 @@ void mapindex_init(void) {
 	int last_index = -1;
 	int index;
 	char map_name[MAP_NAME_LENGTH];
+	char* mapindex_cfgfile[80] = {
+		"db/map_index.txt",
+		"db/import/map_index.txt"
```
Comment (cydh) [inline src/common/mapindex.c]: why don't `db_path` & `DBIMPORT`?

**Comment by cydh** [comment]
then, "how can I remove map by using the import file?" :P

**Comment by Lemongrass3110** [comment]
You can do that in conf/import/map_athena.txt by writing: "delmap: mapname" at least that is ought to work. Did not test to confirm this though.
Deleting something from the cache or index list is unnecassary I guess. You are also able to override data if necessary.

File: src/common/mapindex.c (hunk 1 above)
Comment (Lemongrass3110) [inline src/common/mapindex.c]: As you can see in b5e30e0 it was hardcoded before too. I will adjust this later today. ;)

**Comment by cydh** [comment]
if tested and good, just merge it. I have nothing to test (yet)

**Comment by Lemongrass3110** [comment]
I tested it with Arsinoe from Aerie. :)

**Comment by cydh** [comment]
tested also for __replacing__ only __a__ mapindex or mapcache?

**Pull request merged**